apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1061,6 +1062,14 @@ awaiting_signature(cast, {?SIGNED, settle_tx, SignedTx} = Msg,
                 settle_signed(SignedTx, Updates, D1)
         end, D);
 
+% Timeouts
```

Comment:
A minor quibble, but use two %% at the beginning of a line. The Emacs indentation mode will try to indent a single %

Reply:
Ok, good to know. BTW a quick grep of the codebase ```grep -r -e "^% " apps | wc -l ``` shows that there are 66 occurrences of a single % at the beginning of a line.